unit-testing-react-what-why-how/Basics/problem/package.json

```diff
@@ -8,12 +8,9 @@
   },
   "dependencies": {
```

Comment:
also can't find name `describe` and more. Should we add `types/jest` here to our dependencies?
<img width="960" alt="image" src="https://github.com/bitovi/trainings/assets/65362632/43692f51-5299-4fcb-abd1-3f2db4e4edab">

Reply:
Is that in VS Code? I do not see any warnings when I open up the test files. I’ll be on the lookout though, and please feel free to open a PR to fix it if you figure out the issue.